src/PackageDetailsViewExtension/PackageDetailsViewExtension.cs

```diff
@@ -67,13 +67,7 @@ internal void OpenPackageDetails(PackageManagerSearchElement packageManagerSearc
             if (PackageDetailsView == null) PackageDetailsView = new PackageDetailsView();
             PackageDetailsView.DataContext = PackageDetailsViewModel;
 
-            if (packageManagerSearchElement.UIParent != null)
-            {
-                HostPackageDetailsExtension(packageManagerSearchElement);
-                return;
-            }
-
-            ViewLoadedParamsReference?.AddToExtensionsSideBar(this, PackageDetailsView);
+            HostPackageDetailsExtension(packageManagerSearchElement);
```

Comment:
So we no longer have to check the UI Parent because of the UI refresh of package manager?

Reply:
Yep, reaching this line of code will crash Dynamo, as the UI parent is already attached to a different UI element.